backend/src/models/encounter.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, model } from 'mongoose';
+
+interface EncounterModel {
+  date: Date,
+  location: string,
+  description: string,
+  persons: []
```

Comment:
Might be good to use the `Schema.Types.ObjectId` type for the persons array in the interface